examples/vmod_counters/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use varnish::vsc_wrapper::Vsc;
+use varnish::Stats;
+
+#[derive(Stats)]
+#[repr(C)] // required for correct memory layout
+pub struct VariousStats {
+    /// Number of hits
```

Comment:
perhaps spell out what "hit" means?  Doc that repeats field name is not that useful :)

Reply:
I can make something up 😄 I was struggling to come up with something plausible-seeming that was simple enough that it didn't raise unnecessary red herrings

Reply:
honestly, I don't reckon you need to go for something realistic. I'd be fine with something like:
```
/// Some arbitrary counter
foo: AtomicU64,
```

and in the vmod, have a `.add_one_foo()` function